### DISCUSSION TITLE
UI Settings - Library

### DISCUSSION THREAD

**Discussion by firedfromlife** [gotson/komga#1664] [Ideas]
I'd like to see the following features added as options.
1. Settings to change the default page for a library from Recommended to Browse or Collections.
1. Settings to change the default page size for a library.
1. Display settings, 
    1. Toggle "n books" display.
    1. Sizing options.
    1. Display author.
    
I am aware this is sort of a disjointed wishlist, but it all falls under UX improvements.

**Comment by gotson** [comment]
> Settings to change the default page for a library from Recommended to Browse or Collections.

this is saved in local storage already



> Settings to change the default page size for a library.

this is saved in local storage already



> Display settings,
> 
>     1. Toggle "n books" display.

what does that mean ?

>     2. Sizing options.

what does that mean ?

>     3. Display author.

display where ?

**Comment by firedfromlife** [comment]
- Local storage is not a setting. Meaning using other devices, this does not persist.
- ![image](https://github.com/user-attachments/assets/6c2153c1-b817-4324-b278-feb8319db0f4)
    - That big gray bar beneath each comic, what shows in there should be a setting, or whether it shows at all. As of now it only shows "{Title}" & "{n} books" n being the number of books. You could theoretically have this able to to be toggled, or set to different or additional information, for example the author.
- Sizing options for the thumbnails, even a simple large, medium, small. On a large display, the current size looks kinda odd. Example:
    - ![RNF0-6GzGCwyZls](https://github.com/user-attachments/assets/3302586a-9df4-4a23-9a8a-8ca4dd1c4adf)
    - It looks very odd when there is clearly space if the page size is small.
    - ![3XFK-icRzAC6Fme](https://github.com/user-attachments/assets/aee92ba4-cf78-4f6f-96cf-68f4fc46bb3b)

**Reply by gotson** [reply]
> Local storage is not a setting. Meaning using other devices, this does not persist.

each device most likely need different options anyway, so local storage is a good option

> * That big gray bar beneath each comic, what shows in there should be a setting, or whether it shows at all. As of now it only shows "{Title}" & "{n} books" n being the number of books. You could theoretically have this able to to be toggled, or set to different or additional information, for example the author.

at the moment there's no plans to make this configurable

> * Sizing options for the thumbnails, even a simple large, medium, small. On a large display, the current size looks kinda odd.

debatable, i have a 32" screen, they look alright

**Comment by gotson** [comment]
closing as this is a mixup of various different unrelated requests
